### ISSUE TITLE
'Point of entry' value gets an 'inactive' suffix on point of entry case pages

### ISSUE THREAD

**Issue by adinaflorea9** [hzi-braunschweig/SORMAS-Project#8651]
### Bug Description

On cases that have the 'Case origin' - 'Point of entry', in the section 'Place of stay', the value from the field 'Point of entry' get an inactive suffix added to it once the case is saved.

This is reproducible when creating cases via the 'Cases' directory - 'Create new case form', when creating resulting cases for contacts, event participants and travel entries.

### Steps to Reproduce

1. Open the 'Create new case form';
2. Select the value 'Point of entry' in the section 'Case origin';
3. Input valid information in the mandatory fields;
4. In the section 'Place of stay' select an existing active point of entry (or one of the values other airport/ seaport/ crossing/ entry);
5. Save the case and observe the 'Place of stay' section.

### Actual Behavior

The value listed for the field 'Point of entry' has the 'inactive' suffix added to it.

### Expected Behavior

The value listed for the field 'Point of entry' should not have the 'inactive' suffix added to it (unless the point of entry had been deactivated/archived).

### Screenshots

![image](https://user-images.githubusercontent.com/96909231/161050893-1b9d1d6e-260d-470a-af89-21753b6a060f.png)

![image](https://user-images.githubusercontent.com/96909231/161050913-27ba70dc-140a-42d4-91f8-305f40221d04.png)

### System Details

* Device: Windows 10
* SORMAS version: 1.70.0 and 1.71.0(ddee4c1)
* Android version/Browser: Chrome
* Server URL: test1401.sormas-oegd.de and test-de3.sormas.netzlink.com
* User Role: Admin, NatUser

**Comment by AliciaRaber** [comment]
I tested this issue and have found a possible workaround.
![C_Creation_no_pos](https://user-images.githubusercontent.com/91321821/161722828-48d58f5c-170f-49d0-a6cc-37f532502f4a.png)
I created a case without checking the box place of stay and another case with check on place of stay "Home"
![C_Creation_a_pos](https://user-images.githubusercontent.com/91321821/161722882-0d79e296-2bec-4661-ba2a-5711679f06ef.png)
After case creation the picture shows that the place of stay and entry point are now different.
I don't know if this is a bug and can or should be fixed.
